src/app/my-journeys-listing/my-journey/my-journey.page.html

```diff
@@ -0,0 +1,43 @@
+<app-application-header [config]="headerConfig"></app-application-header>
+
+<ion-content [fullscreen]="true">
+    <div class="journey-name" >
+      <h1  >{{programName}}</h1>
+    </div>
+    <ion-segment class="custom-segment" (ionChange)="segmentChanged($event)" value="inProgress">
+      <ion-segment-button value="inProgress">
+        <ion-label>{{"ONGOING" | translate}}</ion-label>
+      </ion-segment-button>
+      <ion-segment-button value="completed">
+        <ion-label>{{"COMPLETED" | translate}}</ion-label>
+      </ion-segment-button>
+    </ion-segment>
+
+    @if (selectedSegment === 'inProgress') {
+      @if (myJourneyInprogress.length > 0) {
+          <div class="cards-list  layout-wrapper">
+          @for (item of myJourneyInprogress; track $index) {
+            <app-my-journey-ongoing-card [data]="item"></app-my-journey-ongoing-card>
+          }
+        </div>
+        }
+        @else {
```

Comment:
Cant the No_DATA message kept in one place instead of declaring it twice

Reply:
done